packages/rust/src/browser/rust-commands.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { inject, injectable } from 'inversify';
+import { CommandContribution, CommandRegistry, Command } from '@theia/core';
+import { EditorManager, EditorCommands } from '@theia/editor/lib/browser';
+import { NotificationsMessageClient
+       } from '@theia/messages/lib/browser/notifications-message-client';
+import { WorkspaceEdit, Workspace } from '@theia/languages/lib/common';
+import { RustClientContribution } from './rust-client-contribution';
+
+export const SHOW_RUST_REFERENCES: Command = {
```

Comment:
Are you sure you need them? I don't see https://github.com/rust-lang-nursery/rls-vscode/blob/0637e219e21bda2a380fda8e16f14c48b29640a7/src/extension.ts adding these commands